rest-service/manager_rest/rest/resources_v3_1/manager.py

```diff
@@ -201,6 +209,7 @@ def delete(self):
             current_app.logger.info('Manager deleted successfully')
             managers_list = get_storage_manager().list(models.Manager)
             remove_manager(managers_list)  # Removing manager from cluster
+        update_agents(sm)
```

Comment:
it's unfortunate to have this after the db-commit, because this does a LOT of things (amqp connection etc) and so can throw a lot of exceptions.
Maybe we can do something like remove the manager and re-throw if it couldn't be sent. But also what about partially-sent updates...
I dont think we can do anything really sensible here but give it a think

Reply:
We could notify and provide instructions on which endpoint to call to fix it (do we have an endpoint that triggers update-agents?)

Reply:
Update agents to be added to an endpoint+CLI (JIRA needed)